src/utils/salesEvidenceSortAndFilter.ts

```diff
@@ -0,0 +1,44 @@
+// TODO - tsconfig replace CommonJS with esnext
```

Comment:
How'd this get in here?

Reply:
Oh don't worry you merged it in. I would've prefer to have waited to create a PR for that branch into master than merge it directly into this branch.